Forms/Fields/DateOfBirth.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace DUCK.Forms.Fields
+{
+	public struct DateOfBirth
+	{
+		public int day;
+		public int month;
+		public int year;
+
+		public DateOfBirth(int day, int month, int year)
```

Comment:
This should possibly be in the Y/M/D format to match the `DateTime` constructor and try and keep the order consistent. As we found out in a discussion just now though they don't even keep it consistent themselves, and their `TryParse` uses the american M/D/Y format. Either way the ordering is something that is worth keeping in mind to make it less confusing, although I prefer the D/M/Y myself!